app/views/bikes/show.html.haml

```diff
@@ -3,51 +3,52 @@
-      = link_to "#{@bike.type.titleize} in Admin", admin_bike_path(@bike.to_param), class: "btn btn-outline-info uncap", style: "display: block;"
+      = link_to t(".bike_in_admin", bike_type: @bike.type.titleize), admin_bike_path(@bike.to_param), class: "btn btn-outline-info uncap", style: "display: block;"
     - if @bike.user_hidden
       %h2.text-danger.mb-4.text-center
-        HIDDEN by owner
+        = t(".hidden_by_owner")
         %small.less-strong
-          no one but the user (and Bike Index admins) can see it
-  / Authorize check here, even though we're checking in controller too, because otherwise we're displaying user data
+          = t(".no_one_but_the_user_and_bike_index_admins")
+
+  -# Authorize check here, even though we're checking in controller too, because
+  -# otherwise we're displaying user data
   - if current_user.present? && passive_organization.present? && current_user.authorized?(passive_organization)
     = render partial: "organized_access_panel"
   .row
     .col-md-8
       %h1
         - if @bike.stolen
-          - stolen_type = @bike.recovered ? 'recovered' : 'stolen'
           %strong.stolen
-            = "Recovered" if @bike.recovered
-            Stolen
+            = t(".recovered") if @bike.recovered
+            = t(".stolen")
         = @bike.title_u
       - if @bike.stolen && @stolen_record.present?
         %p
           %strong
             - if @bike.recovered
-              Recovered
+              = t(".recovered")
             - else
-              Stolen
+              = t(".stolen")
           %span.convertTime
             = l @stolen_record.date_stolen, format: :convert_time
           %small.convertTimezone
           %strong
-            at
+            = t(".at")
           = @stolen_record.address(skip_default_country: true)
 
       - if @bike.should_show_other_bikes
         %h4
-          = link_to "View user's other registrations", user_path(@bike.owner.username)
+          = link_to t(".view_users_other_registrations"), user_path(@bike.owner.username)
     .col-md-4.ad-col
       .ad-block#top468x60
   .row
-    / Bike Photo!
+    -# Bike Photo!
     .col-md-8
       .bike-photos.horizontal-thumbnails
         - if @bike.public_images.any?
           #selected-photo{ class: "image-holder #{"just1photo" if @bike.public_images.count == 1}" }
             .current-photo{ id: "image#{@bike.public_images.first.id}"}
-              = image_tag(@bike.public_images.first.image_url(:large), alt: "#{@bike.type.titleize} photo", id: "i|#{@bike.public_images.first.listing_order}", data: { action: 'zoom', original: @bike.public_images.first.image_url })
+              = image_tag(@bike.public_images.first.image_url(:large), alt: t(".bike_photo", bike_type: @bike.type.titleize), id: t(".ibikepublic_imagesfirstlisting_order", bikepublic_imagesfirstlisting_order: @bike.public_images.first.listing_order), data: { action: t(".zoom"), original: @bike.public_images.first.image_url })
```

Comment:
this `id` shouldn't have been modified. Fixing in #1016

Reply:
Thanks! Looks like `data` was a spurious translation too